Packs/Legacy/Integrations/Tanium/Tanium.py

```diff
@@ -41,13 +38,52 @@
     raise
 
 
-def csvstr_to_list(str):
-    lines = str.splitlines()
-    if len(lines) < 2:
+def gather_line(lines, total_column_num, current_line):
+    line_values = lines[current_line].split(',')
+    while len(line_values) < total_column_num:
+        current_line += 1
+        line_below = lines[current_line].split(',')
+        line_values[-1] = line_values[-1] + " " + line_below[0]
+
+        if line_values[-1].startswith('\"'):
+            line_values[-1] = line_values[-1][1:]
+
+        if line_values[-1].endswith('\"'):
+            line_values[-1] = line_values[-1][:-1]
+
+        if len(line_below) == 1:
+            continue
+
+        else:
+            line_below = line_below[1:]
+            line_values.extend(line_below)
```

Comment:
not sure what's happening here, please add notes

Reply:
made it clearer in next versions